File: Code/c/pattern8.c
#include<stdio.h>
void main()
{
    int limit, i, j;

    printf("Enter Limit: ");
    scanf("%d", &limit);

    for (i = limit; i > 0; i--)
    {
        for (j = 0; j < limit - i; j++)
            printf("  ");
        for (j = 0; j < 2 * i - 1; j++)
            printf(" *");
        printf("\n");
    }
}

File: Code/java/pattern8.java
import java.util.Scanner;

class Pattern8
{
    public static void main(String[] args)
    {
        int limit, i;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Limit: ");
        limit = scanner.nextInt();

        for (i = limit; i > 0; i--)
            System.out.println("  ".repeat(limit - i) + " *".repeat(2 * i - 1));

        scanner.close();
    }
}